doc/prototype/difference.md

```diff
@@ -6,25 +6,28 @@ in the another set.
 
 @signature `queryLogic.difference(a, b)`
 
-Returns a set that represents the difference of sets _A_ and _B_. In set theory, a difference is
-represented by `A \ B`.
-
-```js
-var queryLogic = new QueryLogic();
-
-
-queryLogic.difference({
-    filter: { name: {$in: ["Ramiya", "Bohdi"]} }
-},{
-    filter: { name: "Bohdi" }
-}))
-//-> {filter: {name: "Ramiya"}}
-
-// A is totally inside B
-queryLogic.difference(
-    {filter: { name: "Bohdi" }},
-    {} )  //-> QueryLogic.EMPTY
-```
+  Returns a set that represents the difference of sets _A_ and _B_. In set theory, a difference is
+  represented by `A \ B`.
```

Comment:
We should give a little more info on what a difference is.  Maybe link to something explaining it.  Even repeating the description helps "Return a query representing the values that are in one set that are not in the another set."